js/service-worker.js

```diff
@@ -197,6 +201,9 @@ class HttpRecorder extends Recorder {
 
   injectForegroundScripts = async (tabId) => {
     super.injectForegroundScripts(tabId);
+    const { manifest_version } = browser.runtime.getManifest();
+
+    console.log({ manifest_version });
```

Comment:
Related to this, I noticed we're doing a `console.log('content script!')` in the content script which shows up in the user's browser console for every page. Can we remove that one and maybe some of these other logs?

Reply:
Thanks! Btw, this shouldn't be displayed on **every** page, only `["*://localhost/*", "https://loadster.app/*", "https://speedway.app/*"]`

Reply:
I'm going to leave the logs in the service worker, most likely the user will not see them

Reply:
Ah, my bad, not much of an issue then.